pkgs/development/python-modules/torchprofile/default.nix

```
@@ -0,0 +1,48 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  setuptools,
+  numpy,
+  torch,
+  torchvision,
+}:
+
+buildPythonPackage rec {
+  pname = "torchprofile";
+  version = "0.0.3";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "zhijian-liu";
+    repo = "torchprofile";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-6vxZHQwBjKpy288wcANdJ9gmvIOZloLv+iN76TtqYAI=";
+  };
+
+  build-system = [
+    setuptools
+  ];
+
+  pythonRelaxDeps = [
+    "torchvision"
+  ];
+
+  dependencies = [
+    numpy
+    torch
+    torchvision
+  ];
+
+  pythonImportsCheck = [
+    "torchprofile"
+  ];
+
+  meta = {
+    changelog = "https://github.com/zhijian-liu/torchprofile/releases/tag/v${version}";
+    description = "A general and accurate MACs / FLOPs profiler for PyTorch models";
```

```suggestion
    description = "General and accurate MACs / FLOPs profiler for PyTorch models";
```
https://nixos.org/manual/nixpkgs/unstable/#var-meta-description

Sorry :')

---

Fixed, thanks !